[2022-08-06 19:36:57,851] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 19:36:57,868] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 19:36:57,869] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:57,869] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:57,869] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:57,885] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 19:36:57,891] {standard_task_runner.py:52} INFO - Started process 980 to run task
[2022-08-06 19:36:57,895] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '286', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpfrh6eb34', '--error-file', '/tmp/tmpzg4yqp10']
[2022-08-06 19:36:57,895] {standard_task_runner.py:80} INFO - Job 286: Subtask second_task
[2022-08-06 19:36:57,968] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:58,058] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 19:36:58,059] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:58,060] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:58,072] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:58,074] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:58,074] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:58,103] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T193657, end_date=20220806T193658
[2022-08-06 19:36:58,146] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:58,178] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:52,650] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 21:50:52,666] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 21:50:52,667] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:52,667] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:52,667] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:52,686] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 21:50:52,694] {standard_task_runner.py:52} INFO - Started process 1180 to run task
[2022-08-06 21:50:52,697] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '287', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpcrluy08i', '--error-file', '/tmp/tmp_8oe_j8r']
[2022-08-06 21:50:52,697] {standard_task_runner.py:80} INFO - Job 287: Subtask second_task
[2022-08-06 21:50:52,821] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:52,948] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 21:50:52,949] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:52,950] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:52,969] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:52,970] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:52,971] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:53,064] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T215052, end_date=20220806T215053
[2022-08-06 21:50:53,110] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:53,162] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:18,560] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 21:56:18,578] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 21:56:18,578] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:18,578] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:18,578] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:18,599] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 21:56:18,609] {standard_task_runner.py:52} INFO - Started process 971 to run task
[2022-08-06 21:56:18,613] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '286', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpcxgi3b83', '--error-file', '/tmp/tmpmklhd2k8']
[2022-08-06 21:56:18,613] {standard_task_runner.py:80} INFO - Job 286: Subtask second_task
[2022-08-06 21:56:18,687] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:18,794] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 21:56:18,795] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:18,795] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:18,813] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:18,815] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:18,815] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:18,859] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T215618, end_date=20220806T215618
[2022-08-06 21:56:18,907] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:18,947] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:18,458] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:11:18,470] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:11:18,471] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:18,471] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:18,471] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:18,484] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 22:11:18,489] {standard_task_runner.py:52} INFO - Started process 970 to run task
[2022-08-06 22:11:18,492] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpatzqqyj1', '--error-file', '/tmp/tmpjdpl58e6']
[2022-08-06 22:11:18,492] {standard_task_runner.py:80} INFO - Job 284: Subtask second_task
[2022-08-06 22:11:18,565] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:18,641] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 22:11:18,642] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:18,643] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:18,654] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:18,656] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:18,656] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:18,682] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T221118, end_date=20220806T221118
[2022-08-06 22:11:18,711] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:18,737] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:19:03,196] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:19:03,210] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:19:03,210] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:03,210] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:19:03,210] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:03,229] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 22:19:03,235] {standard_task_runner.py:52} INFO - Started process 984 to run task
[2022-08-06 22:19:03,238] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpv0vsthqi', '--error-file', '/tmp/tmpo0wc8hk1']
[2022-08-06 22:19:03,238] {standard_task_runner.py:80} INFO - Job 285: Subtask second_task
[2022-08-06 22:19:03,302] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:19:03,381] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 22:19:03,382] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:19:03,383] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:19:03,399] {subprocess.py:85} INFO - Output:
[2022-08-06 22:19:03,401] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:19:03,401] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:19:03,431] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T221903, end_date=20220806T221903
[2022-08-06 22:19:03,449] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:19:03,484] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:42,210] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:22:42,227] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2022-08-06 22:22:42,228] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:42,228] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:42,228] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:42,247] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-31 00:00:00+00:00
[2022-08-06 22:22:42,255] {standard_task_runner.py:52} INFO - Started process 1027 to run task
[2022-08-06 22:22:42,258] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvs8wlnsp', '--error-file', '/tmp/tmp8d5bblzh']
[2022-08-06 22:22:42,258] {standard_task_runner.py:80} INFO - Job 285: Subtask second_task
[2022-08-06 22:22:42,345] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-31T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:42,485] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2022-08-06 22:22:42,487] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:42,488] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:42,508] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:42,510] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:42,511] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:42,613] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211031T000000, start_date=20220806T222242, end_date=20220806T222242
[2022-08-06 22:22:42,673] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:42,731] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
